src/bindgen/parser.rs

```diff
@@ -585,12 +587,21 @@ impl Parse {
         } else {
             warn!("Skip {}::{} - (not `pub`).", crate_name, &item.sig.ident);
         }
-        if (item.sig.abi.is_omitted() || item.sig.abi.is_c()) && !item.is_no_mangle() {
+
+        if !(item.sig.abi.is_omitted() || item.sig.abi.is_c()) {
```

Comment:
we should probably move these warnings above and making them early return instead, but this is fine if you don't want to do that refactoring for now.

Reply:
I'm doing this in a separate change (having a go at #420 - see [here](https://github.com/alaroldai/cbindgen/blob/swift-name/src/bindgen/parser.rs#L618))
I think it makes more sense in that branch because I'm also refactoring this whole function to share functionality with the new `load_syn_method` implementation.